dnf-docker-test/features/install-downgrade-1.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Installing package with lover version
+
+  @setup
+  Scenario: Feature Setup
+      Given repository "available" with packages
+         | Package   | Tag       | Value       |
+         | TestA     | Version   | 1           |
+         |           | Requires  | TestB = 1-1 |
+         | TestA v2  | Version   | 2           |
```

Comment:
Please split this to second repository and then when you test upgrade, enable it